pollination/honeybee_radiance/grid.py

```diff
@@ -28,6 +28,54 @@ def split_grid(self):
     )
 
 
+@dataclass
+class SplitGridFolder(Function):
+    """Create new sensor grids folder with evenly distribute sensors.
+
+    This function creates a new folder with evenly distributed sensor grids. The folder
+    will include a ``_dist_info.json`` file which has the information to recreate the
+    original input files from this folder and the results generated based on the grids
+    in this folder.
+    """
+
+    input_folder = Inputs.folder(
+        description='Input sensor grids folder.',
+        path='input_folder'
+    )
+
+    grid_count = Inputs.int(
+        description='Number of output sensor grids to be created. This number is '
+        'usually equivalent to the number of processes that will be used to run the '
+        'simulations in parallel.', spec={'type': 'integer', 'minimum': 1}
+    )
+
+    sensor_count = Inputs.int(
+        description='Minimum number of sensors in each output grid. Use this number to '
+        'ensure the number of sensors in output grids never gets very small. To ignore '
+        'this limitation set the value to 1. Default: 2000.', default=2000,
```

Comment:
I'm curious about the thought process behind the default `min_sensor_count` of 2000 since it seems really high to me. Weren't we previously using a default `sensor_count` of 200? And I usually found myself dropping it to 30 to get decent parallelization of annual daylight simulations.

If this input actually overrides the `grid_count` (which we should note in the description), shouldn't we set the default value to something closer to the older behavior?

Reply:
This is a very good question and is a hard one to answer. We can talk about it more when we chat today. There is no best number here. As you probably already saw I changed it to 1000 in annual daylight. 200 turned out not to be optimal for the runs on Pollination.